cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -31,10 +33,71 @@
 import static com.powsybl.cgmes.conversion.Conversion.Config.StateProfile.SSH;
 
 /**
+ * TwoWindingsTransformer Interpretation
+ * <p>
+ * Ratio and Phase Interpretation (Xfmr2RatioPhaseInterpretationAlternative) <br>
+ * END1. All tapChangers (ratioTapChanger and phaseTapChanger) are considered at end1 (before transmission impedance) <br>
+ * END2. All tapChangers (ratioTapChanger and phaseTapChanger) are considered at end2 (after transmission impedance) <br>
+ * END1_END2. TapChangers (ratioTapChanger and phaseTapChanger) are considered at the end where they are defined in Cgmes <br>
+ * X. If x1 == 0 all tapChangers (ratioTapChanger and phaseTapChanger) are considered at the end1 otherwise they are considered at end2
+ * <p>
+ * Phase Angle Negate. All phaseTapChanger angles are considered -angle.
+ * <p>
+ * Shunt Admittance Interpretation (Xfmr2ShuntInterpretationAlternative) <br>
+ * END1. All shunt admittances to ground (g, b) at end1 (before transmission impedance) <br>
+ * END2. All shunt admittances to ground (g, b) at end2 (after transmission impedance) <br>
+ * END1_END2. Shunt admittances to ground (g, b) at the end where they are defined in Cgmes model <br>
+ * SPLIT. Split shunt admittances to ground (g, b) between end1 and end2. <br>
+ * <p>
+ * Structural Ratio (Xfmr2StructuralRatioInterpretationAlternative) <br>
+ * END1. Structural ratio always at end1 (before transmission impedance) <br>
+ * END2. Structural ratio always at end2 (after transmission impedance) <br>
+ * X. If x1 == 0 structural ratio at end1, otherwise at end2
```

Comment:
Why ````Xfmr2StructuralRatioInterpretationAlternative```` has a default value to ````X```` and not to ````END2```` ? It is a way to deal with transformers with ````x1 == 0```` and ````x2 != 0```` ?

Reply:
We have seen a TSO that uses this convention in some transformers. 

To obtain  a transformer model with  `ratioTapChager` and `structuralRatio` at side1 (end1) they define the `tapChanger` at the `CgmesTransformerEnd1` and the reactance at the `CgmesTransformerEnd2` (x1 == 0 and x2 != 0). 

To model the `ratioTapChanger` and the `structuralRatio` at side2 (end2), as the rest of TSOs,  they define reactance at `CgmesTransformerEnd1`  (x1 != 0 and x2 == 0) and `tapChanger` at `CgmesTransformerEnd2`

This option (`X`) is more general and also includes the rest of TSOs  that uses the `END2` option.